tests/ballerina-spec-conformance-tests/S05_values_types_variables/tests/implicit_initial_value.bal

```diff
@@ -118,3 +118,27 @@ function testImplicitInitialValueOfJSONType() {
     jsonArray[1] = 3;
     test:assertEquals(jsonArray[0], (), msg = "expected implicit initial value of json type to be ()");
 }
+
+@test:Config {}
+function testArrayMemberTypesBroken() {
```

Comment:
The function name should be renamed. It is not broken anymore :)

Reply:
I think we can remove this and the below test altogether right?
1. They are tested in array test tests anyway.
2. This code only confirm that `QuxObject?` have an implicit initial value, not that QuxObject doesn't have an IIV.

What do you think?